src/test/test_pivx.cpp

```diff
@@ -72,7 +72,7 @@ TestingSetup::TestingSetup()
         pblocktree = new CBlockTreeDB(1 << 20, true);
         pcoinsdbview = new CCoinsViewDB(1 << 23, true);
         pcoinsTip = new CCoinsViewCache(pcoinsdbview);
-        InitBlockIndex();
+        LoadGenesisBlock();
```

Comment:
As now `LoadGenesisBlock()` is returning a result, we can return early in case of an error here.
```
if (!LoadGenesisBlock(chainparams)) {
    throw std::runtime_error("InitBlockIndex failed.");
}

```

Reply:
Actually `InitBlockIndex` was already returning a boolean.
It's `LoadChainTip` that has changed the return value.
But, yeah, we can take the opportunity to add a check here (also, we don't pass `chainparams` yet).

Reply:
Added in 4749d5288990a884ea3b695eb673fb036c87e06a